Us 142 resize able columns edit/delete

tlweave2 opened the pull request:
Added confirmation to delete. made columns resize-able, instituted a reset column button.

tlweave2 committed:
Adjust table widths with .css added table.css

tlweave2 committed:
Added css to between header componets

tlweave2 committed:
Add reset columns button

tlweave2 committed:
Update Table.jsx

The best compromise i could come up with. reset columns resets tabel from previous use of differnt data.

tlweave2 committed:
Update Table.jsx

added confirmation to delete

tlweave2 committed:
Update Unit.test.jsx

qknowles reviewed (changes requested):
Our current version passes the CI. Were you not working with a current version?

tlweave2 committed:
Update Table.jsx

tlweave2 committed:
Update Table.jsx

tlweave2 committed:
Update Table.jsx

tlweave2 committed:
Update Table.jsx

tlweave2 committed:
Update Table.jsx

tlweave2 committed:
Update Table.jsx

back to failed tests

tlweave2 committed:
Update Table.jsx

added css to file in line.

tlweave2 committed:
Update TableHeading.jsx

changed spacing to align columns with headers

tlweave2 committed:
Update TableEntry.jsx

change column spacing to align with headers

tlweave2 committed:
Update Unit.test.jsx

qknowles reviewed (changes requested):
Tim is going to attempt to default the date/time column width. Great job!

tlweave2 committed:
Update Table.jsx

add calculate column to set width in relation to column content width. changed color of slider to appropriate red.

tlweave2 commented:
changed to calculate width of columns from width of contents, changed slider color.
